app/src/main/java/com/libs/globaldata/userdatabase/UserDatabase.java

```diff
@@ -187,6 +212,10 @@ public UserRecord getUserRecord(String userName) {
      * Deserialize user database.
      */
     public void loadDatabase() {
+        if (databaseFile.exists()) {
+            Log.w(Tag + "_" + id, "Unable to load database. File not found");

```

Comment:
return here?

Reply:
Shouldn't it be if (**!**databaseFile.exists())